docs/docs/production.rst

```diff
@@ -63,6 +63,18 @@ Then, produce the model-ready input data with the `produce` function:
 
   result = produce(instance, recipe)
 
+If we want to add to the data few shot examples, we should specify recipe with a data card:
+
+.. code-block:: python
+
+  recipe = "card=cards.wnli,template=templates.classification.multi_class.relation.default,demos_pool_size=5,num_demos=2"
```

Comment:
I don't understand this example. 
Why are you giving the card of wnli? Will it use the example from there?

Reply:
yes the few shot examples come from that card.